Sources/PointFree/PageLayout.swift

```diff
@@ -9,49 +9,88 @@ import Prelude
 import Styleguide
 import Tuple
 
-func respond<A>(_ view: View<A>, layout: @escaping (Flash?, View<A>) -> View<A>)
+struct SimplePageLayoutData<A> {
```

Comment:
this is the layout struct that holds everything the layout needs to do its job. it's already pretty big and I could imagine it getting bigger!